src/hash.h

```diff
@@ -111,8 +111,9 @@ class CHashWriter
     int GetType() const { return nType; }
     int GetVersion() const { return nVersion; }
 
-    void write(const char *pch, size_t size) {
-        ctx.Write((const unsigned char*)pch, size);
+    void write(Span<const std::byte> src)
+    {
+        ctx.Write(UCharCast(src.data()), src.size());
```

Comment:
Any plan to make hash interfaces also use `Span<std::byte>` based arguments?

Reply:
Maybe if I am bored over the holiday season, but not now and not in this pull :sweat_smile: